test/quantization/test_quantize_fx.py

```diff
@@ -3066,29 +3066,44 @@ def test_instance_norm(self):
                 quantized_module, torch.ops.quantized.instance_norm,
                 skip_op_arg_for_functional=True)
 
-    def test_silu(self):
+    def _test_default_node_quant_handler_ops(self, module, functional):
         class M(torch.nn.Module):
-            def __init__(self):
+            def __init__(self, mod, func):
                 super().__init__()
-                self.silu = torch.nn.SiLU()
+                self.module = mod()
+                self.functional = func
 
             def forward(self, x):
-                x = self.silu(x)
-                x = torch.nn.functional.silu(x)
+                x = self.module(x)
+                x = self.functional(x)
                 return x
 
-        data = (torch.randn((2, 2, 2, 2), dtype=torch.float),)
+        data = torch.randn((2, 2, 2, 2), dtype=torch.float16)
         quant_type = QuantType.STATIC
         qconfig_dict = {
             "": float16_static_qconfig
         }
+        m = M(module, functional).eval()
+        m_prep = torch.quantization.quantize_fx.prepare_fx(m, qconfig_dict)
+        m_quant = torch.quantization.quantize_fx.convert_fx(m_prep)
+        self.assertEqual(m_quant(data),m(data.float()), rtol=1e-3, atol=1e-4)
```

Comment:
we don't really need to test this I think, there is not much guarantee for numerics for float vs quantized.

Reply:
perhaps I should add a test, but the intention was to make sure that the model runs correctly with float16 data (the other testing piece doesn't do that), but it sounds like from your other comment that we don't need that?

Reply:
yeah running the model sounds OK. the other tests just makes sure we generate the correct quantized op